configs/stage/permissions/compliance.json

```diff
@@ -3,5 +3,40 @@
         {
             "verb": "*"
         }
+    ],
+    "system": [
+      {
+        "verb": "read"
+      }
+    ],
+    "report": [
+      {
+        "verb": "read"
+      },
+      {
+        "verb": "delete"
+      }
+    ],
+    "policy": [
+      {
+        "verb": "read"
+      },
+      {
+        "verb": "create"
+      },
+      {
+        "verb": "update"
+      },
+      {
+        "verb": "delete"
+      },
+      {
+        "verb": "write"
+      }
+    ],
+    "export": [
```

Comment:
Maybe namespacing the export regarding what are we exporting? Or is there just one thing to be exported from the whole microservice? :thinking: cc @bastilian

Reply:
It covers exporting reports into CSV or JSON and downloading them as a PDF. I've renamed it to `export_report`.

Reply:
`report_export` maybe?